core/src/calleeFunctions/UniswapV2LpTokenCalleeDai.ts

```diff
@@ -25,8 +25,14 @@ const getCalleeData = async function (
         profitAddress,
         joinAdapterAddress,
         minProfit,
-        await getUniswapRouteAddressesBySymbol(network, collateral.exchange.token0),
-        await getUniswapRouteAddressesBySymbol(network, collateral.exchange.token1),
+        await getUniswapRouteAddressesBySymbol(
+            network,
+            (collateral.exchanges['Uniswap Token V2'] as UniswapV2LpTokenCalleeConfig).token0
```

Comment:
Same here and everywhere, please remove those typecasting everywhere by applying the type changes